build/i18n.m4

```diff
@@ -1,16 +0,0 @@
-AC_DEFUN([GP_I18N],
-[
-    if test -z "$conf_dir" ; then
-        conf_dir="."
-    fi
-    ALL_LINGUAS=`cd "$conf_dir/po" 2>/dev/null && ls *.po 2>/dev/null | $AWK 'BEGIN { FS="."; ORS=" " } { print $[1] }'`
```

Comment:
Funnily enough, this looks entirely broken for me on my fresh-ish Debian Bookworm install: I'm not sure where `conf_dir` is supposed to come from, but it's nowhere to be found here, and thus just results in no linguas for out-of-tree builds.  Oopsie, right?  `s/conf_dir/srcdir/` works though, and that's a documented Autoconf variable, whereas I couldn't find references to `conf_dir` in anything current.

Reply:
But this whole file is gone and `LINGUAS` is now checked in, to be updated manually.